packages/automerge-repo-storage-redis/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2024-Present, Ryan Kois
```

Comment:
Looks like in this file you missed the first line. Like [here](https://github.com/facebook/react/blob/64f354cf27a17c4e5b0dfcd908e47940cf947771/LICENSE#L1C1-L1C12).